packages/venia-concept/src/components/Footer/footer.js

```diff
@@ -88,15 +90,21 @@ const Footer = props => {
                             defaultMessage={LOREM_IPSUM}
                         />
                     </p>
-                    <ul className={classes.socialLinks}>
+                    <ul className='footer-socialLinks'>
                         <li>
-                            <Instagram size={20} />
```

Comment:
Should be removed from import (line 3)
PS: In the future, we will use ESLint to detect unused imports.